<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Battleship</class>
 <widget class="QMainWindow" name="Battleship" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>582</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Battleship</string>
  </property>
  <property name="windowIcon" >
   <iconset>pictures/hi48-app-battleship.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="3" >
        <widget class="QPushButton" name="pushButtonQuit" >
         <property name="text" >
          <string>Quit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QPushButton" name="pushButtonStartGame" >
         <property name="text" >
          <string>New Game</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="pushButtonEwo" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="cursor" >
          <cursor>0</cursor>
         </property>
         <property name="text" >
          <string>EWO-Mode</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4" >
        <widget class="QTextEdit" name="console" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="autoFormatting" >
          <set>QTextEdit::AutoBulletList</set>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4" >
        <widget class="QFrame" name="frame" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>3</hsizetype>
           <vsizetype>3</vsizetype>
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>631</width>
           <height>350</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" >
           <widget class="QWidget" name="gridLayoutEnemy" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QWidget" name="gridLayoutHuman" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="font" >
             <font>
              <family>Sans Serif</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text" >
             <string>My Field</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="label_2" >
            <property name="font" >
             <font>
              <family>Sans Serif</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text" >
             <string>Enemy Field</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionManual" />
    <addaction name="actionAbout" />
    <addaction name="actionLicense" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>Options</string>
    </property>
    <addaction name="actionSound_on" />
   </widget>
   <widget class="QMenu" name="menuGame" >
    <property name="title" >
     <string>Game</string>
    </property>
    <widget class="QMenu" name="menuJoin_network_game" >
     <property name="title" >
      <string>Join network game</string>
     </property>
    </widget>
    <addaction name="separator" />
    <addaction name="actionStart_local_game" />
    <addaction name="actionJoin_network_game" />
    <addaction name="actionHost_network_game" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuGame" />
   <addaction name="menuOptions" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>560</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionHost_network_game" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Host network game</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLicense" >
   <property name="text" >
    <string>License</string>
   </property>
  </action>
  <action name="actionJoin_network_game" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Join network game</string>
   </property>
  </action>
  <action name="actionStart_local_game" >
   <property name="text" >
    <string>Single Player</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionManual" >
   <property name="text" >
    <string>Manual</string>
   </property>
  </action>
  <action name="actionSound_on" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Sound on</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>Battleship</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>356</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>Battleship</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>763</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel" >
     <x>796</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStart_local_game</sender>
   <signal>triggered()</signal>
   <receiver>pushButtonStartGame</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>126</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
